src/sequence_statistics.py

```diff
@@ -0,0 +1,229 @@
+# les mots déjà existants sont supprimés
+def generate_new_down(language,length=10):
+    space = requests.get(f'{API_URL}/{language}_syllables/espace/orthographic').json()['result']
+    current = possible(space['following_ipa_syllable'])
+    sentence = ""
+    pronunciation = ""
+    words = []
+    i = 0
+    while i < length:
+        word = ""
+        word_pronunciation = ""
+        while current != " ":
+            try:
+                syll = requests.get(f'{API_URL}/{language}_syllables/{current}').json()['result']
+                word += syll['orthographical_syllable'] + "-"
+                word_pronunciation += syll['ipa_syllable']
+                current = possible(syll['following_ipa_syllable'])
+            except KeyError:
+                print('Problème pour la syllabe : ', current)
+                current = " "
+        if word not in words:
+            existence = requests.get(f'{API_URL}/{language}/{word_pronunciation}/phonetic').json()['result']
+            if existence == []:
+                words += word
+                sentence += word + " "
+                pronunciation += word_pronunciation + " "
+                current = possible(space['following_ipa_syllable'])
+                i += 1
+            else:
+                print('Mot ', existence[0]['spelling'],' (', word_pronunciation, ') retrouvé')
+                current = possible(space['following_ipa_syllable'])
+    print(sentence)
+    print(sentence.replace('-',''))
+    print(pronunciation)
+    return sentence
+
+
+def generate_new_up(language,length=10):
+    space = requests.get(f'{API_URL}/{language}_syllables/espace/orthographic').json()['result']
+    current = possible(space['preceding_ipa_syllable'])
+    sentence = ""
+    pronunciation = ""
+    words = []
+    i = 0
+    while i < length:
+        word = ""
+        word_pronunciation = ""
+        while current != " ":
+            try:
+                syll = requests.get(f'{API_URL}/{language}_syllables/{current}').json()['result']
+                word = syll['orthographical_syllable'] + "-" + word
+                word_pronunciation = syll['ipa_syllable'] + word_pronunciation
+                current = possible(syll['preceding_ipa_syllable'])
+            except KeyError:
+                print('Problème pour la syllabe : ', current)
+                current = " "
+        if word not in words:
+            existence = requests.get(f'{API_URL}/{language}/{word_pronunciation}/phonetic').json()['result']
+            if existence == []:
+                words += word
+                sentence = word + " " + sentence
+                pronunciation = word_pronunciation + " " + pronunciation
+                current = possible(space['preceding_ipa_syllable'])
+                i += 1
+            else:
+                print('Mot ', existence[0]['spelling'],' (', word_pronunciation, ') retrouvé')
+                current = possible(space['preceding_ipa_syllable'])
+    print(sentence)
```

Comment:
this is for debug only, so i would remove them.
also better use logging than prints.